qiita_db/metadata_template/constants.py

```diff
@@ -70,6 +70,15 @@
 ALL_RESTRICTIONS = [SAMPLE_TEMPLATE_COLUMNS, PREP_TEMPLATE_COLUMNS,
                     PREP_TEMPLATE_COLUMNS_TARGET_GENE]
 
+# This is what we consider as "NaN" cell values on metadata import
+NA_VALS = ['', 'no_data', 'unknown', 'Unspecified']
```

Comment:
`NA_VALS` -> `NAN_VALS`

@biocore/qiita-dev in my opinion, N/A has a different meaning in metadata and we should keep it (i.e. is not the same having a null, no_data, unknown, and Unspecified that an N/A). Hence the name change here.

Can you also add 'unspecified'?